storage/localstate_mgr.go

```diff
@@ -52,9 +60,14 @@ func NewFileStateMgr(folder string) (*FileStateMgr, error) {
 			}
 		}
 	}
+	key, err := getFragmentSeed()
```

Comment:
The key is always encrypted?
Maybe we could keep the option to leave it unencrypted for testing purposes?

Reply:
The key is just a string in an environment variable, then it is hashed. The key will be clear if you get the env var.

Reply:
ZetaClient fails to start if `TSS_FRAGMENT_SEED` is not defined? We should ensure observers can still run without having the env var defined